test/steps_test.dart

```diff
@@ -9,4 +10,35 @@ void main() {
         FeatureFile(featureDir: '$path.feature', package: path, input: '');
     expect(feature.getStepFiles().length, 0);
   });
+
+  test('When step with examples then param take column name', () {
```

Comment:
Isn't it a duplicate of `Scenario Outline Step` from `test/scenario_outline_step_test.dart`? If you agree — please remove this test.